inst/examples/03-formats.Rmd

```diff
@@ -218,9 +231,235 @@ favicon: "favicon.ico"
+```yaml
+bookdown::bs4_book:
+  theme:
+    primary: "#0d6efd"   
+    base_font: 
+      google: Sen
+    heading_font:
+      google:
+        family: Bitter
+        wght: 200
+    code_font:
+      google: 
+        # arguments to sass::font_google() 
+        family: DM Mono
+        local: false
+```
+
+#### Callout blocks
+
+The `bs4_book` theme also includes special callout blocks with predefined styles, which will include a colored border around the text and/or code inside the callout. Use the following syntax to create a callout block: 
+
+````markdown
+::: {.rmdnote}
+The `bs4_book` theme also includes an `.rmdnote` callout block like this one.
+```{r collapse=TRUE}
+head(beaver1, n = 5)
+```
+:::
+````
+
+You may use Markdown syntax and inline code inside a block. When knitted, this will render as:
+
+```{r fig.align='center', echo=FALSE, include=identical(knitr:::pandoc_to(), 'html'), fig.alt="A special rmd-note callout block."}
+knitr::include_graphics('images/rmd-note.png', dpi = NA)
+```
+
+Available blocks are: `.rmdnote`, `.rmdcaution`, `.rmdimportant`, `.rmdtip`, and `.rmdwarning`. The colors used will be based on the default colors provided by Bootstrap, but can be also be customized in your `_output.yml` file:
+
+```yaml
+bookdown::bs4_book:
+  theme:
+    primary: "#0d6efd"   # default .rmdnote = blue
+    danger:  "#dc3545"   # default .rmdcaution = red
+    success: "#198754"   # default .rmdimportant = green
+    info:    "#0dcaf0"   # default .rmdtip = cyan
+    warning: "#ffc107"   # default .rmdwarning = yellow
+```
+
+For LaTeX output, only the content of these blocks will be shown with no colored outline as for HTML. It is up to the user to define the appearance of these blocks for LaTeX output using custom environments. See this section in The R Markdown Cookbook: <https://bookdown.org/yihui/rmarkdown-cookbook/custom-blocks.html> for a how-to.
+
+#### HTML metadata
+
+Bookdown will generate HTML `<meta>` tags based on Pandoc's variables set in `index.Rmd`, described in \@ref(metadata-for-sharing). Additionally, `bs4_book()` will create unique chapter descriptions auto-generated from the chapter's contents. You can have a look at [`bs4_book()` HTML
+template](https://github.com/rstudio/bookdown/blob/main/inst/templates/bs4_book.html) for details on how these variables are used.
+
+#### References/Bibliography
+
+The `bs4_book` format makes footnotes appear inline when you click on them. 
+Making your citations _footnotes_
+allows readers to read them near the text they are referred in.
```

Comment:
```suggestion
Making your citations _footnotes_ allows readers to read them near the text where they're used because `bs4_book` makes footnotes appear inline when clicked. 
```